portfolio.html

```diff
@@ -43,133 +57,152 @@ <h1 class="item2">I'm a software developer</h1>
     </div>
     </main>
 
-<!-- the second part of the project starts form here -->
+   <!-- the second part of the project starts form here -->
     
-   <section class="grid_container_1">
-    <div>
+  <section class="grid_container_1">
+    <div class="imp-div">
       <h1 class="second_page_title">My Recent Works</h1>
       <hr class="divider">
     </div>
       
    <div class="card-container">
-    <div class="card-1">
-      <img src="Img Plaholder@1x.png" alt="project image">
-    </div>
-      
-    <div class="card-2">
-
-      <div class="title"><h3>Multi-Post Stories Gain+Glory</h3></div>
+    <div class='project-card'>
+      <div class="project-title">
+        <h3>Multi-Post Stories Gain+Glory 
+        </h3>
+      </div>
       
-      <ul class="abilites">
+      <ul class="project-abilites">
         <li class="a1">Ruby on rails</li>
         <li class="a1">css</li>
         <li class="a1">JavScript</li>
         <li class="a1">html</li>          
-      </ul>
-
-      <div class="b_div"><button class="button">See Project</button></div>
```

Comment:
- [ ] Please, kindly fix the vertical spacing between these cards, to enable a very pixel-perfect design, like you'll be doing in your forthcoming job 😀.

- Your design
<img width="279" alt="wrong" src="https://user-images.githubusercontent.com/41163571/221199539-391f7468-1291-44e6-8c6e-3c1674d52300.png">

- Figma design
<img width="265" alt="right" src="https://user-images.githubusercontent.com/41163571/221199608-8ad517ec-5e33-49b2-9f51-d222837006ec.png">